docs/tutorial/plotting_images.rst

```diff
@@ -0,0 +1,187 @@
+Plotting Images
+===============
+
+.. only:: html
+
+    :nbexport:`Download this page as a Jupyter notebook <self>`
+
+This tutorial covers the built-in plotting functionality for images in Pylake. While most of the
+examples shown use kymographs, the details described here are valid for
+:meth:`Kymo.plot()<lumicks.pylake.kymo.Kymo.plot>`, :meth:`Scan.plot()<lumicks.pylake.scan.Scan.plot>`
+and :meth:`ImageStack.plot()<lumicks.pylake.ImageStack.plot>`.
+
+We'll start with loading some data as usual::
+
+    import lumicks.pylake as lk
+    filenames = lk.download_from_doi("10.5281/zenodo.7729525", "test_data")
+
+    file = lk.File("test_data/kymo.h5")
+    print(file.kymos)  # dict of available kymos: {'16': Kymo(pixels=699)}
+
+We can use the :meth:`~lumicks.pylake.kymo.Kymo.plot()` convenience function to plot individual
+color channels and the full RGB image::
+
+    plt.figure(figsize=(6.4, 7))
+
+    # plot just the green channel
+    plt.subplot(3, 1, 1)
+    kymo.plot("green", adjustment=lk.ColorAdjustment(0, 15), aspect="auto")
+
+    #plot just the red and green channels
+    plt.subplot(3, 1, 2)
+    kymo.plot(adjustment=lk.ColorAdjustment(0, 99, mode="percentile"), aspect="auto")
```

Comment:
Forgot to actually pass the color channel argument (this will plot all channels).
```suggestion
    kymo.plot("rg", adjustment=lk.ColorAdjustment(0, 99, mode="percentile"), aspect="auto")
```